<launch>
    <arg name="aruco_maker_size" default="0.053">
    <group ns="cam1_disp">
        <include file="$(find disp_pkg)/launch/displacement.launch">
            <arg name="camera" value="/camera_1"/>
            <arg name="image" value="image"/>
            <arg name="fiducial_len" value="$(arg aruco_marker_size)"/>
            <arg name="dictionary" value="7"/>
        </include>
    </group>
    <group ns="cam2_disp">
        <include file="$(find disp_pkg)/launch/displacement.launch">
            <arg name="camera" value="/camera_2"/>
            <arg name="image" value="image"/>
            <arg name="fiducial_len" value="$(arg aruco_marker_size)"/>
            <arg name="dictionary" value="7"/>
        </include>
    </group>
    <group ns="cam3_disp">
        <include file="$(find disp_pkg)/launch/displacement.launch">
            <arg name="camera" value="/camera_3"/>
            <arg name="image" value="image"/>
            <arg name="fiducial_len" value="$(arg aruco_marker_size)"/>
            <arg name="dictionary" value="7"/>
        </include>
    </group>
</launch>